device-types/Dell/EMC-MD1400.yml

```diff
@@ -0,0 +1,40 @@
+---
+manufacturer: Dell
+model: EMC MD1400
+slug: dell-emc-md1400
+part_number: emc-md1400
+front_image: true
+rear_image: true
+comments: |
+  [Dell EMC MD1400 Datasheet](https://i.dell.com/sites/csdocuments/Shared-Content_data-Sheets_Documents/en/DellStorage_MD1400_MD1420_082114.pdf)
+u_height: 2
+is_full_depth: true
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 600
+module-bays:
+  - name: Slot 1
+    position: '1'
+  - name: Slot 2
+    position: '2'
+  - name: Slot 3
+    position: '3'
+  - name: Slot 4
+    position: '4'
+  - name: Slot 5
+    position: '5'
+  - name: Slot 6
+    position: '6'
+  - name: Slot 7
+    position: '7'
+  - name: Slot 8
+    position: '8'
+  - name: Slot 9
+    position: '9'
+  - name: Slot 10
+    position: '10'
+  - name: Slot 11
+    position: '11'
+  - name: Slot 12
+    position: '12'
```

Comment:
Are these all for storage media? If so, Storage media is not currently supported, please remove these definitions